### PR TITLE
[PLAYER-3813] [RECEIVER] Update cast.js to allow all player param from receiver page

### PR DESCRIPTION
**Pull request by DmitriiMaslov**
This PR updates default page level parameters in receiver template. 
We need this to verify ticket https://jira.corp.ooyala.com/browse/PLAYER-3813 and ensure that page level parameters can be passed to player V4 through the receiver.

### CODE REVIEW

File: lib/template.html [hunk 1]
```diff
@@ -38,10 +38,19 @@
             playerBrandingId: "pleaseFillTheRightPlayerId",
             skin: {
                 inline: {
-                    pauseScreen: {
-                        showDescription: false
-                    }
-                }
+					startScreen: {
```
Comment (dallen42) [inline lib/template.html]: @DmitriiMaslov can we do away with these extra layers of indents here to keep the formatting clear?

**Comment by pilievOoyala** [comment]
LGTM aside from the indentation.

File: lib/template.html (hunk 1 above)
Comment (DmitriiMaslov) [reply lib/template.html]: Formatting corrected

**Comment by aldigun** [comment]
@DmitriiMaslov @KseniiaSokolovaEpam what is the point of this merge? Are we really trying to hide the Title and Description of the asset on receiver UI? If this is for testing, we should not attempt this with code merge. You should be able to make a testing receiver build from a BRANCH by using sandbox environment.

**Comment by DmitriiMaslov** [comment]
@aldigun this was your point (see comments in ticket for QA) to test the changes with staging receiver ID. The fix was tested with test app id in a sand box.

**Comment by DmitriiMaslov** [comment]
As it was agreed we don't need to merge these changes into Master so PR can be closed.

**Pull request closed without merging**